### ISSUE TITLE
[Docker] Opening configuration file /etc/apt/apt.conf.d/01autoremove - ifstream::ifstream (13: Permission denied)

### ISSUE THREAD

**Issue by brylie** [CiviWiki/OpenCiviWiki#512]
When following the Docker installation steps, I get the following error when running `docker-compose up`:

```
...
Step 5/17 : RUN apt-get update && apt-get install -y --no-install-recommends   git   gcc   vim   build-essential   unzip   xvfb   libxi6   libgconf-2-4   default-jdk   supervisor &&   apt-get clean &&   rm -rf /var/lib/apt/lists/*
 ---> Running in 1734b3eb694f
E: Opening configuration file /etc/apt/apt.conf.d/01autoremove - ifstream::ifstream (13: Permission denied)
ERROR: Service 'backend' failed to build: The command '/bin/sh -c apt-get update && apt-get install -y --no-install-recommends   git   gcc   vim   build-essential   unzip   xvfb   libxi6   libgconf-2-4   default-jdk   supervisor &&   apt-get clean &&   rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
```

**Comment by TomaszKolek** [comment]
@brylie I think it's your platform-specific error. I haven't met it on any of my 3 PCs, I believe others also haven't. Were you able to google it?

**Comment by brylie** [comment]
@TomaszKolek isn't step five RUN command executing inside of the image?

**Comment by TomaszKolek** [comment]
The command is relatively simple - is just package installation - do you have any proxy set on your PC or something?

I rebuilt it a few times and it still works.

**Comment by brylie** [comment]
Ah, I may have had a work proxy active. I'll try again.

**Comment by brylie** [comment]
Hm, I tried again with just normal network connection and still get:

> ERROR: Service 'backend' failed to build: The command '/bin/sh -c apt-get update && apt-get install -y --no-install-recommends   git   gcc   vim   build-essential   unzip   xvfb   libxi6   libgconf-2-4   default-jdk   supervisor &&   apt-get clean &&   rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100

**Comment by brylie** [comment]
Might this be related to HTTPS?

https://stackoverflow.com/a/38004106/1191545

@richifuentes has also been asking about HTTPS support in a different issue:
https://github.com/CiviWiki/OpenCiviWiki/issues/229#issuecomment-494427286

**Comment by brylie** [comment]
Hm, probably not related to HTTPS or networking at all, as it explicitly says it's a permission issue:

> E: Opening configuration file /etc/apt/apt.conf.d/01autoremove - ifstream::ifstream (13: Permission denied)

**Comment by brylie** [comment]
OK, it turns out that Docker doesn't have an official release yet for Ubuntu 19.04. I am able to build the image after enabling a testing Docker repository.

**Comment by Manvi07** [comment]
Hi @brylie 
Recently when I tried building a docker image, I got the same error. I am new to docker-compose. Can you please help me to fix this? I am using MANJARO Linux 19.0.2.
